lib/global_uid.rb

```diff
@@ -27,3 +27,8 @@ class TimeoutException < Exception ; end
 if ActiveRecord::VERSION::MAJOR >= 4
   ActiveRecord::SchemaDumper.send(:prepend, GlobalUid::SchemaDumperExtension)
 end
+
+# Make sure that GlobalUID is disabled for ActiveRecord's Internal Metadata table
+if ActiveRecord::VERSION::MAJOR >= 5
+  ActiveRecord::InternalMetadata.global_uid_disabled
```

Comment:
Thinking about it, I'd rather keep it this way. I prefer having an explicit error if that class ends up not being defined than having something failing down the line.